man/systemd-cryptenroll.xml

```diff
@@ -387,6 +387,28 @@
         enrollment to.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--tpm2-external-key=</option><replaceable>PATH</replaceable></term>
```

Comment:
so this explanation here does not mention the word SRK at all, but this is almost certainly going to be the SRK of the target machine, no? Maybe say it like that.

Reply:
yep, i reworded this to clarify it's the srk.